packages/base/Form/src/FormField/styles.ts

```diff
@@ -64,60 +47,14 @@ export const createLabelWidthStyles = (
   )
 }
 
-export const createStylesForHorizontalLayout = (theme: Theme) => {
-  return {
-    display: 'grid',
-    // --form-label-width is passed down from cascading style, in this case from Form
-    '--label-width': `calc(${HORISONTAL_LABEL_COLUMN_WIDTH} / 4 * var(${FORM_LABEL_WIDTH_CSS_VARIABLE}, ${DEFAULT_LABEL_WIDTH_SIZE}))`,
-    gridTemplateColumns: `var(--label-width) 1fr`,
-    gap: '0 32px', // 0 and lg, respectively
-    gridTemplateRows: 'auto auto',
-    gridTemplateAreas: `
-        "label input"
-        "hint error"
-      `,
-    width: '100%',
-
-    // create media queries for each breakpoint to set --form-label-width
-    ...createBreakpointsForLabelWidth(theme),
-  }
-}
-
-export default (theme: Theme) =>
-  createStyles({
-    root: {
-      fontSize: '1rem',
-
-      '& + &': {
-        marginTop: '1em',
-      },
-      '& $error + $hint': {
-        marginTop: 0,
-      },
-    },
-
-    adornment: {
-      position: 'relative',
-      paddingRight: '2rem',
-    },
-    autoSaveIndicator: {
-      position: 'absolute',
-      top: 0,
-      right: 0,
-
-      '&$hasMultilineCounter': {
-        top: '-0.875rem',
-      },
-    },
-
-    horizontalLayout: createStylesForHorizontalLayout(theme),
-
-    horizontalLayoutAdornment: {
-      gridArea: 'error',
-    },
-
-    // These classes might still be used in selectors
-    hasMultilineCounter: {},
-    hint: {},
-    error: {},
-  })
+export const horizontalLayoutClasses = `
+  grid w-full gap-x-[32px] gap-y-0 grid-rows-[auto_auto] 
+  grid-cols-[calc((17rem/4)*var(--form-label-width,3))_1fr]
```

Comment:
```suggestion
  grid-cols-[calc(4.25rem*var(--form-label-width,3))_1fr]
```
as it is static now, we can just calculate it right away